app/views/browse/service.html

```diff
@@ -100,7 +100,7 @@ <h3>Pods</h3>
                 <annotations annotations="service.metadata.annotations"></annotations>
               </div>
             </uib-tab>
-            <uib-tab active="selectedTab.events" ng-if="'events' | canI : 'watch'">
+            <uib-tab active="selectedTab.events" ng-if="eventsVersion | canI : 'watch'">
```

Comment:
Maybe I missed it, but I don't see where `$scope.eventsVersion` is set?

Reply:
bah, my bad.  thx, will update & retest.

Reply:
I still don't see `eventsVersion` set?

Reply:
push your changes Ben... sorry, didn't push that last update.  This looks right now.